naughtystrings/naughtystrings.go

```diff
@@ -0,0 +1,37 @@
+// Package naughtystrings is a collection of strings that have a high probability of causing issues when used as user input.
+package naughtystrings
+
+import (
+	"encoding/json"
+	"os"
+	"path"
+	"runtime"
+)
+
+// Base64Encoded returns the strings encoded in base 64.
+func Base64Encoded() []string {
+	return strings("blns.base64.json")
+}
+
+// Unencoded returns the strings.
+func Unencoded() []string {
+	return strings("blns.json")
+}
+
+func strings(file string) []string {
+	var _, here, _, _ = runtime.Caller(0)
+
+	there, err := os.Open(path.Join(path.Dir(path.Dir(here)), file))
+
+	if err != nil {
+		panic(err)
```

Comment:
You don't run this code only on init but every time you run `Unencoded` or `Base64Encoded`. Here error can be from opening file (and it's always error prone...) and not programmer error.